app/SequenceModel.hs

```diff
@@ -0,0 +1,81 @@
+-- Model fitting to derive coefficients in
+-- Math.NumberTheory.Primes.Sequence.chooseAlgorithm
+
+module Main where
+
+import Numeric.GSL.Fitting
+
+-- | Benchmarks Sequence/filterIsPrime
+filterIsPrimeBenchData :: [([Double], ([Double], Double))]
+filterIsPrimeBenchData =
+  [ ([100000, 1000], ([777], 0.1))
```

Comment:
Some comments explaining what these numbers mean would be useful. Just a short description.

Reply:
Done.